lawlibrary/templates/lawlibrary/home.html

```diff
@@ -46,8 +46,17 @@ <h4 class="mb-4">
 {% endblock %}
 
 {% block first-column-content %}
-  <h2 class="mb-4 heading-underlined">{% trans 'Recent Judgments' %}</h2>
-      {% include 'peachjam/_recent_document_list.html' with documents=recent_judgments view_more_text='View more Judgements' view_more_url_name='judgment_list' %}
+
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent Judgments' %}</h2>
+    {% include 'peachjam/_recent_document_list.html' with documents=recent_judgments view_more_text='View more Judgements' view_more_url_name='judgment_list' %}
+  </section>
+
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent National Legislation' %}</h2>
```

Comment:
The title says national but the data is everything. Perhaps just drop National from the title